src/components/Button/Button.styled.ts

```diff
@@ -0,0 +1,38 @@
+import { styled, css } from 'styled-components'
+import { COLOR_VARIANTS, FONT_VARIANTS } from './Button.types';
+
+const fontBasic = css`
+  font-family: Lato, sans-serif;
```

Comment:
GlobalStyles add `Lato` as default font

Reply:
i think you can add this styles to `GlobalStyle` file

Reply:
Instruction moved to `GlobalStyles` like so:

```css
button {
  font-family: ${({ theme }) => theme.fonts.lato};
  font-size: 18px;
}
```